Minio.Examples/Program.cs

```diff
@@ -184,10 +184,13 @@ public static void Main(string[] args)
                 // Delete the object
                 Cases.RemoveObject.Run(minioClient, destBucketName, objectName).Wait();
 
+                // Tacing request with custom logger
+                Cases.CustomRequestLogger.Run(minioClient).Wait();
```

Comment:
This example can be commented out so that output is clean
s/Tacing/Tracing

Reply:
Sorry, I didn't see this comment early